aperturedb/cli/transact.py

```diff
@@ -0,0 +1,11 @@
+import json
+
+import typer
+
+app = typer.Typer()
+
+
+@app.command()
+def from_json_file(filepath: str):
```

Comment:
for the use case of creating indexes, we can just put the query on a .json file and use this. neat. 

I am still unsure what should go on Python scripts and what to move to bash. I think we will learn over time, but if you have an idea, a Readme with your practices to learn from would be good.